Source/OxyPlot/Series/BoxPlotSeries.cs

```diff
@@ -130,6 +145,11 @@ public BoxPlotSeries()
         public bool ShowMedianAsDot { get; set; }
 
         /// <summary>
+        /// Gets or sets a value indicating whether to show the mean as a dot.
+        /// </summary>
+        public bool ShowMeanAsDot { get; set; }
```

Comment:
Yeah I thought about that, but I didn't want to introduce breaking changes for this, but if you think it's better to change this I can implement a enum as well ;)